js/templates/modals/listingDetail/listing.html

```diff
@@ -2,8 +2,19 @@
 
 <div class="modalContentInner">
   <div class="listingControls flex">
+    <div class="jsModalClose contentBox clrP clrBr clrSh3" style="height: 35px; line-height: 35px; padding: 0 8px; cursor: pointer">
+      <div class="userIcon floL discTn clrBr2 clrSh1"
+        <% var avatarHash = ob.avatarHashes ? ob.isHiRez() ? ob.avatarHashes.small : ob.avatarHashes.tiny : ''; %>
+        <% if (avatarHash) { %>
+          style="background-image: url(<%= ob.getServerUrl(`ipfs/${avatarHash}`) %>), url('../imgs/defaultAvatar.png')"
+        <% } else { %>
+         style="background-image: url('../imgs/defaultAvatar.png');"
+        <% } %>>
+      </div>
+      <div style="overflow: auto; font-size: 13px">Store Name <u style="font-size: 12px">View store</u></div>
```

Comment:
No, we only have data from the contract API, which doesn't return the name of the node that owns the listing. 

We'd have to either add it when the listing modal is created (if the modal was opened from the store page), or make a profile call to get it using the guid (the listing does have the guid of the seller). 

Calling the profile would be needed when the listing was opened from a channel or search results, since in that case the client wouldn't already have the profile information for the seller.

(btw, a quick way to find out what data is available in a template is to add <% console.log(ob) %>somewhere, that will list all the functions and data you can access in the console).